android/src/main/java/com/chirag/RNMail/RNMailModule.java

```diff
@@ -22,10 +25,24 @@
 public class RNMailModule extends ReactContextBaseJavaModule {
 
   ReactApplicationContext reactContext;
+  private static final int MAIL_SEND_REQUEST = 10101;  
+  Callback callback;
+  
+  private final ActivityEventListener mActivityEventListener = new BaseActivityEventListener() {
+    @Override
+    public void onActivityResult(Activity activity, int requestCode, int resultCode, Intent intent) {
+      if (requestCode == MAIL_SEND_REQUEST) {
+          if(callback != null)
+            callback.invoke("mailclient_closed");
```

Comment:
this only tells that you came back from the email sending activity. doesn't necessarily mean you actually sent the email right?

Reply:
That is right. Either user cancel / send the email.